beacon-chain/blockchain/log.go

```diff
@@ -21,7 +22,17 @@ func logStateTransitionData(b *ethpb.BeaconBlock) {
 		"attesterSlashings": len(b.Body.AttesterSlashings),
 		"proposerSlashings": len(b.Body.ProposerSlashings),
 		"voluntaryExits":    len(b.Body.VoluntaryExits),
-	}).Info("Finished applying state transition")
+	}).Debug("Finished applying state transition")
```

Comment:
I prefer the usage of `Info` for this over `Synced`. Seeing how many beacon operations are in the block is very nice imo

Reply:
+1

Reply:
I can restore `INFO` level here, sure. It will just create some redundancy.